Client/Assets/ContextMenu.js

```diff
@@ -102,7 +101,6 @@ addEventListener("touchstart", function (event) {
   // Also listen for mouseup due to touchend not being fired with VoiceOver enabled.
   // Open bug filed as rdar://22256909.
```

Comment:
Delete these two lines since they're no longer relevant after you remove the `mouseup` listener.